projects/core/src/lib/colors/color-map-selector/mpl-colormaps.ts

```diff
@@ -11,12 +11,12 @@ import {RgbaTuple} from '../color';
 /**
  * The color map names.
  */
-export type MplColormapName = 'MAGMA' | 'INFERNO' | 'PLASMA' | 'VIRIDIS';
+export type MplColormapName = 'MAGMA' | 'INFERNO' | 'PLASMA' | 'VIRIDIS' | 'GRAYS';
```

Comment:
is "grays" the correct name in mpl? I only find "gray". Also they are all in lower case in mpl 🤔

Reply:
i think it is "Grays" in the MPL source: https://github.com/matplotlib/matplotlib/blob/main/lib/matplotlib/_cm.py